src/components/Migrate/Converter/ConverterFormControls.tsx

```diff
@@ -169,41 +217,37 @@ function ConverterFormControls({
             ${validationStatus === 'valid' ? `color ${theme.accent};` : ''}
           `}
         >
-          {parseFloat(formattedAmount) * CONVERSION_RATE}
+          {parseFloat(formattedAmount) * conversionRate}
         </span>{' '}
         ANTv2
       </p>
-      <Info
-        css={`
-          margin-top: ${3 * GU}px;
-          margin-bottom: ${2 * GU}px;
-        `}
-      >
-        This conversion is a one way path.{' '}
-        <Link
-          href={antV2ContractUrl}
-          css={`
-            display: inline-flex;
-            align-items: center;
-            text-decoration: none;
-            line-height: 1;
-          `}
+
+      {showError && !allowanceCheckLoading && (
+        <ValidationWarning status={validationStatus} />
+      )}
+
+      <ConversionInfo />
+      {conversionType === 'ANJ-LOCK' && (
+        <div
```

Comment:
not blocking but concern: we have mixed use of react styling, CSS rewiring, and styled-component. we need to reach a decision soon on this matter

Reply:
yeah, this is an old project that I copied the original styling. In fact most V1 web apps use these type of styling, including Aragon UI, this Upgrade app, the v1 client app, etc.... but, yeah, totally agree to be consistent on the use of React styling in newer projects, like Govern.  I'm prepared to remove the CSS rewiring stuff I introduced to Govern if that's the consensus.